drivers/crypto/Kconfig

```diff
@@ -55,7 +55,7 @@ source "drivers/crypto/Kconfig.stm32"
 source "drivers/crypto/Kconfig.nrf_ecb"
 source "drivers/crypto/Kconfig.intel"
 source "drivers/crypto/Kconfig.npcx"
-source "drivers/crypto/Kconfig.xec"
+source "drivers/crypto/Kconfig.mchp"
```

Comment:
This commit must be the 2nd in the list.
It still a preparation for what is coming next.